### PR TITLE
gh-116932: Add notes on how to report python documentation theme bugs

### LINKED ISSUE #116932: docs (easy): Add reference to the Python docs theme gh repo
The paragraph should be added in bugs.html
```
If you find a bug in the theme (HTML / CSS / JavaScript) of the
documentation, please submit a bug report on the `python-doc-theme bug
tracker <https://github.com/python/python-docs-theme>`_.
```

_Originally posted by @JulienPalard in https://github.com/python/python-docs-theme/issues/21#issuecomment-411396446_

### Linked PRs
* gh-117989
* gh-118031
* gh-118040
* gh-118041

### PR DESCRIPTION
**Pull request by lyc8503**
Fix #116932

* Issue: gh-116932

----
📚 Documentation preview 📚: https://cpython-previews--117989.org.readthedocs.build/

### CODE REVIEW

**Review by encukou — APPROVED** [review]

**Comment by encukou** [comment]
Thank you!

File: Misc/NEWS.d/next/Documentation/2024-04-17-22-16-19.gh-issue-116932.lETyxa.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Add notes on how to report python documentation theme bugs
```
Comment (Privat33r-dev) [inline Misc/NEWS.d/next/Documentation/2024-04-17-22-16-19.gh-issue-116932.lETyxa.rst:1]: News entry is [not needed](https://devguide.python.org/core-developers/committing/#changes-that-require-news-entries). But, I guess, it's a bit late now for this comment.

**Comment by Privat33r-dev** [comment]
@AlexWaygood should the news entry be reverted?

**Comment by AlexWaygood** [comment]
> @AlexWaygood should the news entry be reverted?

Meh. I probably wouldn't have included a news entry for this either, but I don't think it's the biggest issue in the world to have a single additional entry in an already-very-long changelog :-)

If you put up a PR to remove the news entry, I'd probably merge it... but I don't know that it would be the _best_ use of your time ;-)

**Comment by lyc8503** [comment]
Sorry I didn't notice this didn't need a NEWS entry, I'll file a PR to
remove it.

Alex Waygood ***@***.***> 于 2024年4月18日周四 下午5:21写道：

**Pull request merged**